script.js

```diff
@@ -0,0 +1,96 @@
+//Вивести в console числа, від 10 до 25 з кроком 0.5 (10 й 25 включно)
+
+let i = 10;
+while (i <= 25) {
+  console.log(i);
+  i += 0.5;
+}
+
+// Запитати та зберегти число введене із prompt.
+//Визначати чи це число просте
+
+const enteredNumber = +prompt(`Please, enter any number.`);
+let isPrimeNum = `The number ${enteredNumber} is Prime.`;
+for (let i = 2; i < enteredNumber; i++) {
+  if (enteredNumber % i === 0) {
+    isPrimeNum = `The number ${enteredNumber} is't Prime.`;
+  }
+}
+console.log(isPrimeNum);
+
+//Запитати та зберегти суму покупки введене із prompt. Обрахувати знижку та повідомити користувачу значення знижки. Якщо сума покупки до 100 UAН - знижка 3%; якщо сума покупки до 200 UAH - знижка 5%; якщо сума покупки перевищує 200 UAH - знижка 7%
+
+const howMuchMony = +prompt(`What is the sum of your purchase?`);
+let discount = 0;
+for (let i = 0; i <= howMuchMony; i++) {
```

Comment:
And so on, fix the names
```suggestion
for (let i = 0; i <= howMuchMoney; i++) {
```